### PR TITLE
[PluginsGet] - Update/Fix.

### PR DESCRIPTION
**Pull request by ivandrofly**
- Restore cursor to default when download operation fails after setting cursor to wait.
- Remove `Cursor = Cursors` after a async operation.
- Add helper method to avoid code duplication.

### CODE REVIEW

File: src/Forms/PluginsGet.cs [hunk 1]
```diff
@@ -377,14 +375,10 @@ private void buttonUpdateAll_Click(object sender, EventArgs e)
                     wc.DownloadDataCompleted += wc_DownloadDataCompleted;
                     wc.DownloadDataAsync(new Uri(_updateAllListUrls[i]));
                 }
-                Cursor = Cursors.Default;
```
Comment (ivandrofly) [inline src/Forms/PluginsGet.cs]: Shouldn't be here... it will be called just after the line # 378 finishes creating async task.

**Comment by ivandrofly** [comment]
Plugins update seems to have a bug... WIP.
(Plugins)[http://i.imgur.com/Mo1cIl7.gifv]

**Comment by niksedk** [comment]
@ivandrofly: Ready?

**Comment by ivandrofly** [comment]
yeah this one is ready. I'm currently on bug mentioned above @niksedk

**Comment by niksedk** [comment]
I cannot re-create the issue above - works fine here...

**Comment by ivandrofly** [comment]
that is weird, I tried from both VS and latest release (portable) it still not updating the plugins after clicking **Update All** button ... I'll try on another pc to see if problem still exist.

**Comment by ivandrofly** [comment]
the issue was being caused due to un-updated (binary file) (LinesUnbreaker). fixed :)

**Pull request merged**